OBJ Reader/headers/OBJReader.h

```diff
@@ -0,0 +1,11 @@
+#pragma once

```

Comment:
Instead of #pragma once do this :-
 
#ifndef OBJREADER_H
#define OBJREADER_H
----- Your Code -----
#endif